terraform/shared/modules/grafana/ecs-security.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_security_group" "ecs_task" {
+  name        = "grafana-ecs-tasks"
+  description = "For Grafana tasks, access from LB and EFS only"
+  vpc_id      = var.vpc_id
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_security_group_rule" "ecs_tasks_ingress" {
+  type                     = "ingress"
+  protocol                 = "tcp"
+  from_port                = 3000
+  to_port                  = 3000
+  source_security_group_id = aws_security_group.lb.id
+  description              = "ECS task ingress rule, allow access from LB"
+  security_group_id        = aws_security_group.ecs_task.id
+}
+
+#tfsec:ignore:aws-ec2-no-public-egress-sgr
```

Comment:
Why do we need this egress? Is this to get to prometheus?

Reply:
Yep